rust/examples/mobile-client.rs

```diff
@@ -181,12 +181,11 @@ fn perform_task(
         Err((client, _)) => client,
     };
 
-    match client.get_global_model().unwrap() {
-        Some(model) => println!(
+    if let Some(model) = client.get_global_model().unwrap() {
```

Comment:
we can have `if let` with an `else` clause now? :exploding_head: This is cool

Reply:
i think omitting the `else` clause of the `if let` only works because the `if` clause has the `!` return type. i guess it won't work in a variable assignment.